### ISSUE TITLE
[Feature] Add conversations model

### ISSUE THREAD

**Issue by ADikmarov** [fivetran/dbt_hubspot#126]
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Describe the Feature

https://developers.hubspot.com/docs/api/conversations/conversations

Information about conversations for users is critical to us (HubSpot Live Chat / WhatsApp / etc.). 
It would be really cool to add it to the HubSpot dbt

### Are you interested in contributing this feature?

- [ ] Yes.
- [X] Yes, but I will need assistance and will schedule time during your [office hours](https://calendly.com/fivetran-solutions-team/fivetran-solutions-team-office-hours) for guidance.
- [ ] No.

**Comment by fivetran-jamie** [comment]
hey there! 

Unfortunately i don't think the Fivetran Hubspot connector supports Conversations yet, but I did come across this [Feature Request](https://support.fivetran.com/hc/en-us/community/posts/8862338430871-Connector-Improvement-Hubspot-Conversations-inbox-and-messages-APIs) from a year ago. If you'd like to chime in and connect with our data PM, perhaps we can get some movement on adding Conversations to the connector, which would need to happen prior to the package supporting them.

**Comment by fivetran-joemarkiewicz** [comment]
Marking this as won't fix until the relevant updates are applied in the connector.
